ext/hash/hash_murmur.c

```diff
@@ -82,9 +93,24 @@ const php_hash_ops php_hash_murmur3c_ops = {
 	0
 };
 
-PHP_HASH_API void PHP_MURMUR3CInit(PHP_MURMUR3C_CTX *ctx)
+PHP_HASH_API void PHP_MURMUR3CInit(PHP_MURMUR3C_CTX *ctx, HashTable *args)
 {
-	memset(&ctx->h, 0, sizeof ctx->h);
+	if (args) {
+		zval *seed;
+		/* This might be a bit too restrictive, but thinking that a seed might be set
+			once and for all, it should be done a clean way. */
+		if ((NULL != (seed = zend_hash_str_find(args, "seed", sizeof("seed") - 1))) && IS_LONG == Z_TYPE_P(seed)) {
```

Comment:
Similarly here, _deref is needed to make it work with a seed that happens to be a reference.

Reply:
Fixed, thanks.